acoustic_msgs/msg/SonarImage.msg

```diff
@@ -32,17 +55,20 @@ float32 azimuth_beamwidth
 # elevation direction, in radians
 float32 elevation_beamwidth
 
-# Azimuth for center of each beam in radians. Zero bearing is straight
-# ahead with positive to starboard or up, depending on the orientation
-# of the sensor
+# Azimuth for center of each beam in radians
+# Unless otherwise specified, zero azimuth is assumed to 
```

Comment:
I don't like "unless otherwise specified" comments, because there's nowhere to programmatically specify additional info.

Zero azimuth *will* be along the sensor's X-axis, by definition, which is usually straight ahead from the sensor.